src/Core/src/Handlers/Button/ButtonHandler.Android.cs

```diff
@@ -62,28 +66,37 @@ protected override void DisconnectHandler(MaterialButton platformView)
 
 			ImageSourceLoader.Reset();
 
+			_defaultDrawable = null;
+
 			base.DisconnectHandler(platformView);
 		}
 
 		// This is a Android-specific mapping
 		public static void MapBackground(IButtonHandler handler, IButton button)
 		{
-			handler.PlatformView?.UpdateBackground(button);
+			if (handler is ButtonHandler buttonHandler)
```

Comment:
I don't think we need to case the handler. The PlatformView on IButtonHandler is MaterialButton

https://github.com/dotnet/maui/blob/847a7af88ec5c48e780748dbc834285cb810dfc0/src/Core/src/Handlers/Button/IButtonHandler.cs#L4

Reply:
Right, but require the cast to `ButtonHandler `to access to the default drawable and pass it updating the Background.